<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSslChat Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="hostnameLabel">
        <property name="text">
         <string>Hostname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hostnameLineEdit">
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="portSpinBox">
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>32767</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connectDisconnectButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="chatDisplayTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="inputLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>connectDisconnectButton</sender>
   <signal>clicked()</signal>
   <receiver>Client</receiver>
   <slot>connectDisconnectButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>Client</receiver>
   <slot>sendButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectDisconnectButtonPressed()</slot>
  <slot>sendButtonPressed()</slot>
 </slots>
</ui>
